docs/searching-algorithms/linear-search.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Linear Search
+slug: /searching-algorithms/linear-search
+---
+
+---
+title: Linear Search
+slug: /searching-algorithms/linear-search
+---
+
+# Introduction
+
+Linear search is the simplest of the search algorithms. The motive of any searching algorithm is to search for the presence of a key element in the collection of elements( arrays, linked lists, etc). Linear search starts the search of the key element(the element whose presence in the given list is to be searched) sequentially, from the beginning to the list until it is found. 
```

Comment:
Keep it `search of the key element (the element whose presence in the given list is to be searched) sequentially`

Reply:
Done